### DISCUSSION TITLE
How to use private key from TPM from Podman 4.2

### DISCUSSION THREAD

**Discussion by kishorebhadri** [containers/podman#19286] [General]
Problem statement -

Podman needs to pull an image from a private registry using certificate and the private key stored in the TPM.

Question -
Is there is feature that is implemented in Podman version 4.2 to use the private key that is in TPM. if so can the documentation on how to configure in Podman can be shared.

Else which version of Podman supports TPM.

**Comment by rhatdan** [comment]
Podman has no interface working with the TPM that I know of.  Someone might have written a secret plugin which could work with podman login --secret, perhaps.
